runtime/tests/runtime_integration_tests/test_boot.rs

```diff
@@ -75,3 +77,30 @@ fn test_update() {
     assert_eq!(fw_rev[0], 0xaaaaaaaa);
     assert_eq!(fw_rev[1], 0xaabbccdd);
 }
+
+#[test]
+fn test_boot_tci_data() {
+    let mut model = run_rt_test(Some(&firmware::runtime_tests::MBOX), None, None);
+
+    let rt_journey_pcr_resp = model.mailbox_execute(0x1000_0000, &[]).unwrap().unwrap();
+    let rt_journey_pcr: &[u8] = rt_journey_pcr_resp.as_bytes();
+
+    let valid_pauser_hash_resp = model.mailbox_execute(0x2000_0000, &[]).unwrap().unwrap();
+    let valid_pauser_hash: &[u8] = valid_pauser_hash_resp.as_bytes();
+
+    const MEASUREMENT_SIZE: usize = DPE_PROFILE.get_hash_size();
+    let mut boot_tci_data = [0u8; 3 * MEASUREMENT_SIZE];
+    boot_tci_data[MEASUREMENT_SIZE..2 * MEASUREMENT_SIZE].copy_from_slice(rt_journey_pcr);
+    boot_tci_data[2 * MEASUREMENT_SIZE..].copy_from_slice(valid_pauser_hash);
+    let hashed_boot_tci_data = model
+        .compute_sha512_acc_digest(&boot_tci_data, ShaAccMode::Sha384Stream)
+        .unwrap();
```

Comment:
I think this might be more clear if you use the https://docs.rs/sha2/latest/sha2/ crate and call update for each measurement, instead of using the accelerator.

Reply:
It looks like there's some differences in the caliptra-internal sha384 and the normal sha384. I tried hashing some simple things like [0u8; 48] and "hello world" and the sha2::Sha384 hash and the caliptra sha384 hash differed. I ended up creating a new mbox cmd in mbox_responder that reads data from the mailbox, hashes it, and writes it back to the mbox so that we can use the same hasher that caliptra uses in the tests.

Reply:
>  I tried hashing some simple things like [0u8; 48] and "hello world" and the sha2::Sha384 hash and the caliptra sha384 hash differed

Huh, that's unexpected. We should maybe look into why that is.